website/docs/guides/getting_started.html.markdown

```diff
@@ -205,11 +205,35 @@ resource "newrelic_alert_channel" "slack" {
 This should avoid any of the configuration items from causing a change to the
 resource.
 
+## Add a Label to the Application
```

Comment:
Adding this to the getting started guide is great.  Can you also add a specific resource document? Good example is the [Synthetics Label](https://github.com/terraform-providers/terraform-provider-newrelic/blob/master/website/docs/r/synthetics_label.html.markdown) doc.

Reply:
Thanks for the comment. I removed it from Getting started and moved to the dedicated resource as suggested.